src/test/java/BeforeAfter.java

```diff
@@ -0,0 +1,23 @@
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class BeforeAfter {
+    public WebDriver driver;
```

Comment:
Lines 9-10: Why are they public?

Reply:
It's my bad. I just didn't pay attention.
Now it is corrected.